contracts/core/FlowFees.cdc

```diff
@@ -0,0 +1,131 @@
+import FungibleToken from "./FungibleToken.cdc"
```

Comment:
Add a comment including repository and commit for this contract

Reply:
Could you elaborate?

Reply:
All copied contracts should have a reference to the repository where they came from, for example:
https://github.com/DapperCollectives/VESSEL/blob/main/contracts/core/BloctoToken.cdc#L3